### ISSUE TITLE
running scripts with new route 1 and route 7

### ISSUE THREAD

**Issue by nquisen** [GOLewis-McChord/go_src#6]
Looks like there are some issues with the how the kml and django routes are being processes for route 1 and route 7.  I just did a fresh git pull which updated the stop.kml file.  Validation results attached.

**Comment by mlockwood** [comment]
I am not seeing validation results or error messages. Did something not attach?

**Comment by nquisen** [comment]
Oops, didn't see the message saying html wasn't an accepted file type.  Here you go.  Thanks.
[validation-results (2).zip](https://github.com/GOLewis-McChord/go_src/files/1001783/validation-results.2.zip)

**Comment by mlockwood** [comment]
Did someone put shapes 39 and 40 into shapes.txt? That would make that first set of errors go away. The timetravel one is on the planning side in software. Perhaps a time is off on the data entry for Route 1 or Route 7? I would have Nathan review.

**Comment by nquisen** [comment]
Shapes 39 and 40 are new this morning after doing the git pull.  Assuming Nathan updated the stops.kml file.  Last week before I did the git pull the only shape error was for shape 20.  Background: Nathan gave me a kml file for route 1.  It's dual line work meaning it has the line for the entire route which puts 2 offset lines on roads where the route goes both directions on it.  I took the kml file brought it into GIS to replace our existing route 1 file and then rexported the entire available 1 routes and put the  file in the appropriate folder.  Did not manually edit shapes.txt file.

**Comment by mlockwood** [comment]
When the shapes get updated you still have to run the shape_kml.py file on the latest KML to produce an updated shapes.kml with 39 and 40. That should have been part of master.py but it is possible that the step was commented out. Do you know if it is running?

The time travel will still be an issue with the times selected on the template. I could probably figure it out looking over the times in the Django templates. However that would be overstepping my bounds as this is the planner's domain.

But if I understand correctly -- scripts are running to completion without errors. We are just seeing validation errors from the inputs into the process?

**Comment by nquisen** [comment]
I was never able to get Master.py to run successfully so I run the pieces individually. I run GTFS.py and shape_KML.py.  I can't remember why I run them separately since shape_kml.py is called at the end of GTFS.py.  Previously these ran without error however with the recent change to route 1 both tell me  "Shape 0 does not have a matching segment" repeated for 8 lines and then exit code 0.  Then I run route.py with no errors.

I told Nathan about looking at the times in Django but haven't heard back from him yet.

**Comment by nquisen** [comment]
I'm out of the office for the rest of the day but will look at message first thing in the morning.  Thanks for looking at this.

**Comment by mlockwood** [comment]
Roger. If I remember correctly that error message is not a serious one and is more of a log statement than an actual error. Try running shape_kml.py again and then gtfs.py again. Let me know if that at least clears the error with a missing shape for 39 and 40.

**Comment by nquisen** [comment]
Just ran shape_kml.py followed by gtfs.py. Feedvalidator has the same errors as before.  shape 39 and 40.  Does the stops.kml file under the src/static/kml folder affect this? I thought that was just for the visual mapping but when I did a "git pull" and that file got updated the shape error went from be 20 to 39 and 40.  Nathan and I were thinking that the stops get updated from the django route segments.
